components/plan-form/PlanForm.js

```diff
@@ -24,8 +44,8 @@ export default function PlanForm() {
     }
 
     return (
-        <section>
-            <form onSubmit={(event) => handleSave(event)}>
+        <FormContainer>
+            <FormCreatePlan onSubmit={(event) => handleSave(event)}>
```

Comment:
could be shortened to:

```suggestion
            <FormCreatePlan onSubmit={handleSave}>
```

however, the function does nothing for the moment, but prevent the default behaviour. Does it serve a purpose yet?

Reply:
I did something else there before but had to remove it since it's not part of the user story. Didn't remove all the parts I guess.